### PR TITLE
chore: update uuid

### PR DESCRIPTION
**Pull request by zhennann**

### CODE REVIEW

**Review by stephenscholz — APPROVED** [review]

**Comment by alexandrebodin** [comment]
Would love to get this merged and released. Can we help somehow @dead-horse?

**Comment by fengmk2** [comment]
I will publish a minor version before all tests pass.

**Comment by fengmk2** [comment]
6.3.0

File: package.json [hunk 1]
```diff
@@ -23,16 +23,16 @@
...
+    "supertest": "^3.3.0",
+    "uid-safe": "^2.1.3"
   },
   "license": "MIT",
   "dependencies": {
-    "crc": "^3.4.4",
-    "debug": "^3.1.0",
-    "is-type-of": "^1.0.0",
-    "uuid": "^3.3.2"
+    "crc": "^4.0.0",
```
Comment (fengmk2) [inline package.json:32]:
```
peerDependencies WARNING crc@^4.0.0 requires a peer of buffer@>=6.0.3 but none was installed
```

I will downgrade crc to v3

Comment (fengmk2) [reply package.json:32]: https://github.com/koajs/session/pull/223

**Pull request merged**